### ISSUE TITLE
Mario Kart 8 Deluxe crash

### ISSUE THREAD

**Issue by coolllman** [Ryujinx/Ryujinx#4349]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of Issue

If vsync off in ryujinx (and vsync on in nvidia control), when in-game go to gran-pri game crash

### Reproduction Steps

Disable vsync in ryujinx and go to gran pri mode

### Log File

[Ryujinx_1.1.596_2023-01-29_09-29-09.log](https://github.com/Ryujinx/Ryujinx/files/10528884/Ryujinx_1.1.596_2023-01-29_09-29-09.log)

### OS

Windows 10

### Ryujinx version

1.1.596

### Game version

2.2.1

### Environment?

- ##### CPU: *(e.g. i7-11700)*
- ##### GPU: *(e.g. NVIDIA RTX 3060)*
- ##### RAM: *(e.g. 32GB)*
- Applied Mods: [ No ]

**Comment by AcK77** [comment]
`Applied Mods: [ No ]` accordingly to you log file, you use a mod for 60FPS. Does it crash without the mod?

**Comment by coolllman** [comment]
Yes its crash without 60fps mod too, ryujinx vsync very jagger in mario kart.

**Comment by moznr542** [comment]
This worked correctly in builds up to ryujinx-1.1.561. ryujinx-1.1.562+ crashes when entering gran pri mode with vsync off.

**Comment by coolllman** [comment]
Yes, i test 1.1.561 and all works fine, regression in build 1.1.562

**Comment by Silent-Remux** [comment]
Can confirm this happens to me consistently.

**Comment by riperiperi** [comment]
I don't think https://github.com/Ryujinx/Ryujinx/pull/4294 can affect this, as it only affects the games list, not presentation. Current recommendation is to just enable vsync.

**Comment by Silent-Remux** [comment]
> I don't think #4294 can affect this, as it only affects the games list, not presentation. Current recommendation is to just enable vsync.

Bad solution when Ryujinx's vsync in fundamentally broken.

**Comment by riperiperi** [comment]
It is not fundamentally broken, it works in terms that the game expects. A game crashing when you completely break the rules is more fundamentally broken.

**Comment by coolllman** [comment]
With ryujinx vsync i have jagger, and its not solution too fix this, and its regression((

**Comment by Silent-Remux** [comment]
In 1.1.650, with vsync turned off after the grand prix select screen, after the 4 races it crashes as you press "view results".

**Comment by yogithesymbian** [comment]
> Yes its crash without 60fps mod too, ryujinx vsync very jagger in mario kart. Log without mod: [Ryujinx_1.1.605_2023-02-08_14-43-30.log](https://github.com/Ryujinx/Ryujinx/files/10683508/Ryujinx_1.1.605_2023-02-08_14-43-30.log)

hey , can you share 60 fps mod ? i stuck on 23 - 29 fps on grandprix . only smooth on time trials 30 - 39 fps 
( the bad things for me on time trial, we cant auto acceleration , only on grandprix we can do ).
